test/misc/misc_test.go

```diff
@@ -69,31 +68,3 @@ func TestMisc_integration(t *testing.T) {
 		}
 	})
 }
-
-func TestMisc_connection_error(t *testing.T) {
```

Comment:
these tests could be readded?

Reply:
Does it make sense to readd this? I feel like this case is covered by WaitForWeaviate.

Reply:
as long as we have the ready checker in the code base, we should have a test for it